data/mods/Magiclysm/Spells/druid.json

```diff
@@ -207,5 +207,27 @@
     "spell_class": "DRUID",
     "energy_source": "MANA",
     "flags": [ "PERMANENT", "NO_LEGS", "CONCENTRATE" ]
+  },
+  {
+    "id": "purify_seed",
+    "type": "SPELL",
+    "name": "Purification Seed",
+    "description": "You summon a gift of the earth which will purify water. Greater levels yield greater numbers of seeds.",
+    "valid_targets": [ "self" ],
+    "min_damage": 1,
+    "max_damage": 6,
+    "damage_increment": 0.25,
+    "effect": "spawn_item",
+    "effect_str": "druid_seed",
+    "base_casting_time": 600,
+    "base_energy_cost": 800,
+    "final_energy_cost": 800,
+    "min_duration": 1,
+    "max_duration": 1,
+    "difficulty": 8,
+    "max_level": 20,
+    "spell_class": "DRUID",
+    "energy_source": "MANA",
+    "flags": [ "PERMANENT", "NO_LEGS", "CONCENTRATE" ]
```

Comment:
permanent does not mean the item summoned is always permanent. permanent means at max level the spell will be permanent. what this spell does is from level 0-19 wastes all of your mana because it spawns an item that despawns immediately. I suggest that rather than using that flag, give a reasonable duration to the spell. the player's going to be using the item immediately anyway.

Reply:
alright. i fixed that aspect and added more max duration, giving the ability to purify a little more than a gallon at max level @ the cost of 3k mana. also adjusted difficulty down somewhat.

Reply:
duration is in moves so you need to multiply those by 100 if you want anything meaningful there.

Reply:
alright. i changed around durations and casting times. item should last 1 minute across the board, casting time should begin at 5 minutes and drop to 1 minute at max level. mana usage is now a flat 1k.

item now uses transform script that i found in grenades and creates another item with 15 charges (only works as a non-comestible with charges), which will allow you to purify up to a gallon of water. the duration carries over.

for whatever reason if i just created an item with charges it would just linger with 0/15 charges til the duration timed out, but the transform script makes the item disappear after use regardless of whether or not it used 1 or 15 of the charges to purify.

Reply:
oh, that casting time was supposed to be 5 minutes? casting time is in moves, not turns.

Reply:
the newest commit should be 5 minutes, unless i fucked up my math

Reply:
also if there's a way to edit this specific commit without going into desktop and pushing an entirely new commit to address changes.. I don't know how to do it. I'm pretty ignorant about github and the etiquette involved. appreciate the patience as I puzzle it out.

Reply:
![image](https://user-images.githubusercontent.com/29068879/61134606-f63c4600-a48d-11e9-895f-bcaf0b407b92.png)
You have to be a little careful not to mess up the linting, but changing one little value is pretty simple.